tests/sentry/monitors/tasks/test_detect_broken_monitor_envs.py

```diff
@@ -562,14 +566,29 @@ def test_sends_emails_to_owners_team_id(self, mock_now, builder):
         monitor, monitor_environment = self.create_monitor_and_env()
         team_member1 = self.create_user("teammember1@example.com")
         team_member2 = self.create_user("teammember2@example.com")
```

Comment:
Are these verified by default?

Reply:
ahh yes, verified by default based on a signal https://github.com/getsentry/sentry/blob/fd0d63d99f3d27c85b97111c4026d48bb1d79c06/src/sentry/testutils/factories.py#L829-L830

Reply:
ahh i guess i can also write a test, to filter out non-verified emails one sec